net/upnp/src/www/services_upnp.php

```diff
@@ -171,6 +178,8 @@ function miniupnpd_validate_port($port)
         foreach (['enable', 'enable_upnp', 'enable_natpmp', 'logpackets', 'sysuptime', 'permdefault'] as $fieldname) {
             $upnp[$fieldname] = !empty($pconfig[$fieldname]);
         }
+        // numeric types
+        $upnp['num_permuser'] = $pconfig['num_permuser'];
```

Comment:
only set when offered `!empty()` and add a validation if the parameter provided is actually a valid number (the `$input_errors[] =` blocks above).

Reply:
Done.